src/Controls/src/SourceGen/CodeBehindGenerator.cs

```diff
@@ -130,10 +168,25 @@ static string EscapeIdentifier(string identifier)
 			return null;
 		}
 
+		var allowImplicitXmlns = itemAndCaches.Item2.AllowImplicitXmlns;
+
+		var nsmgr = new XmlNamespaceManager(new NameTable());
+		nsmgr.AddNamespace("__f__", XamlParser.MauiUri);
+		nsmgr.AddNamespace("__g__", XamlParser.MauiGlobalUri);
+		if (allowImplicitXmlns)
+		{
+			nsmgr.AddNamespace("", XamlParser.DefaultImplicitUri);
+			foreach (var xmlnsPrefix in xmlnsCache.XmlnsPrefixes)
+				nsmgr.AddNamespace(xmlnsPrefix.Prefix, xmlnsPrefix.XmlNamespace);
+		}
+		using var reader = XmlReader.Create(new StringReader(text.ToString()),
+											new XmlReaderSettings { ConformanceLevel = allowImplicitXmlns ? ConformanceLevel.Fragment : ConformanceLevel.Document },
+											new XmlParserContext(nsmgr.NameTable, nsmgr, null, XmlSpace.None));
```

Comment:
This is a non-trivial piece of code and there are quite a few copies of it here and in the XamlC code. Would it make sense to move it to a separate method for consistency sake?